README.md

```diff
@@ -12,6 +12,26 @@
 * `npm install`
 * `node ./server/create-lb-tables.js`
 * `node ./server/migrate.js`
+* Add a .env file to the root directory with the following:
+```
+SESSION_SECRET=anylongstringoftext
+COOKIE_SECRET=anotherlongstringoftext
+```
+* If you want to support OAuth with Facebook, Twitter and/or GitHub you will need to add consumer client ids and secrets to .env
+```
+FACEBOOK_APP_ID={grab from app at https://developers.facebook.com/apps/214528186128969)
+FACEBOOK_APP_SECRET={grab from app at https://developers.facebook.com/apps/214528186128969}
+TWITTER_CONSUMER_KEY={grab from app at https://developer.twitter.com/en/apps/15974878}
+TWITTER_CONSUMER_SECRET={grab from app at https://developer.twitter.com/en/apps/15974878}
+GITHUB_CLIENT_ID={grab from https://github.com/settings/applications/944507}
+GITHUB_CLIENT_SECRET={grab from https://github.com/settings/applications/944507}
```

Comment:
These urls are only accessible if you have the correct permissions. Who is keeping the accounts? Can you give the creds to adam if you have no already done so?

Reply:
Yeah. Adam gave me the permissions so he already does. I think he needs to add you but I can try